app/Models/KnowledgeCafe/Library/Book.php

```diff
@@ -23,21 +22,30 @@ public function categories()
 
     public static function getList($filteredString = false)
     {
-        return self::with(['categories', 'readers', 'borrowers'])
+        $query = self::with(['categories', 'readers', 'borrowers']);
+        return $query
             ->where(function ($query) use ($filteredString) {
                 if ($filteredString) {
                     $query->where('title', 'LIKE', "%$filteredString%")
                         ->orWhere('author', 'LIKE', "%$filteredString%")
                         ->orWhere('isbn', 'LIKE', "%$filteredString%");
                 }
-
-                ($filteredString) ?: '';
             })
             ->withCount('readers')
             ->orderBy('readers_count', 'desc')
             ->get();
     }
 
+    public function scopeKindle($query)
+    {
+        return $query->where('on_kindle', true);
+    }
+
+    public function scopeExcludeKindle($query)
```

Comment:
I think a better name for this would be 
```suggestion
    public function scopeNotOnKindle($query)
```